lib/cdo/shared_constants/pd/shared_workshop_constants.rb

```diff
@@ -0,0 +1,65 @@
+module Pd
```

Comment:
Is this generated from the other worshop_constants file? If so, should this be checked in?

Reply:
No, this is the source of the shared constants. It's [included](https://github.com/code-dot-org/code-dot-org/pull/21689/files#diff-89d6ee82adc13d605c8e55c8185f89d4R5) in `Pd::WorkshopConstants`, which also contains non-shared Ruby-only constants, and it [generates](https://github.com/code-dot-org/code-dot-org/pull/21689/files#diff-b36dc43ff8a3ac75fd15cf6bda24d0b1R90) a javascript `/apps/src/generated/pd/sharedWorkshopConstants.js` file which is not checked in. In fact everything under `apps/src/generated` is [gitignored](https://github.com/code-dot-org/code-dot-org/blob/staging/.gitignore#L27).